docs/deployment.rst

```diff
@@ -9,6 +9,52 @@ Caching
 
 There is a plenty of options you can use to create proper caching infrastructure: Varnish, CloudFlare, ...
 
+Cloudflare Cache Rules
+-----------
+
+Cloudflare supports custom rules for configuring caching:
+https://developers.cloudflare.com/cache/about/cache-rules/
+
+You may want to cache default ``.pbf`` and ``.json`` responses. Create a rule which matches ``hostname (equal)`` and ``URI Path (ends with)`` for both of the .pbf and .json fields. Set cache status to eligible for cache to enable the caching and overwrite the ``Edge TTL`` with ``Browser TTL`` to be 7 days (depends on your application usage).
```

Comment:
there could also be [png, jpg (jpeg), webp](https://maptiler-tileserver.readthedocs.io/en/latest/endpoints.html#rendered-tiles) at the raster endpoints.

Reply:
Any recommendation on why going for one option than another ?

What about memcache and mapProxy ? Any benefit compared to nginx cache ?

Reply:
I'd say one thing about mapProxy is it doesn't seem to support vector tiles (pbf files), which I'd say is one of the main focuses in tilesever.  It would support the rendered tiles, but not vector tiles out of the box. This solution seems like it would support both rendered tiles and vector tiles.

Reply:
@acalcutt updated.

Reply:
Accidently commited with another user.

Reply:
@acalcutt Did not try mapProxy so cannot compare the benefits. For me its easier to manage Nginx reverse proxy. I assume mapProxy has more capabilities to customize caching based on the tiles rendered. While Nginx caches on file (object) level.

Reply:
Apparently with mapproxy you can save a lot for identical tiles, like for ocean raster tiles

Reply:
To avoid a warning in nginx error log, add the following lines :
 ###### Set as shown below. You can use other values for the numbers as you wish
     proxy_headers_hash_max_size 512;
     proxy_headers_hash_bucket_size 128;